atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTopologyValidator.java

```diff
@@ -212,8 +232,17 @@ private Set<CassandraServer> getNewHostsWithInconsistentTopologiesFromTopologyRe
                 // between refreshes for legitimate reasons (but they should still refer to the same underlying
                 // cluster).
                 if (pastConsistentTopology.get() == null) {
-                    // We don't have a record of what worked in the past, so just reject.
-                    return newServersWithoutSoftFailures.keySet();
+                    ClusterTopologyResult result =
```

Comment:
also note for future: We don't need this (and likely don't want this!) when we have this PR in https://github.com/palantir/atlasdb/pull/6753, since we'll just end up checking the same config again :D